module/Company/src/Company/Model/CompanyPacket.php

```diff
@@ -3,16 +3,14 @@
 namespace Company\Model;
 
 use Doctrine\ORM\Mapping as ORM;
-//use Doctrine\Common\Collections\ArrayCollection;
-//use Zend\Permissions\Acl\Role\RoleInterface;
-//use Zend\Permissions\Acl\Resource\ResourceInterface;
+use Doctrine\Common\Collections\ArrayCollection as ArrayCollection;
 
 /**
  * CompanyPacket model.
  *
  * @ORM\Entity
  */
-class CompanyPacket //implements RoleInterface, ResourceInterface
+class CompanyPacket
```

Comment:
While I agree that that this should be named a package, this should've been addressed in #41 already. Now it is a much bigger hassle to fix.